Sources/Kitura/TypeAliases.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright IBM Corporation 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import KituraContracts
+
+/*
+  This file declares public typealiases for types stored in the dependencies.
+  The purpose is to expose these types at the top level without having to import a specific dependency.
+*/
```

Comment:
Have you tested what Jazzy does with this file? Does it look correct? (I've had instances where successive comment blocks with no code separating them causes the documentation to vanish from the next code block).